horovod/run/common/util/env.py

```diff
@@ -45,3 +45,11 @@ def get_env_rank_and_size():
     # Default to rank zero and size one if there are no environment variables
     return 0, 1
 
+
+def is_mpi_job():
```

Comment:
I would suggest renaming to `is_kubeflow_mpi` for clarity.

Reply:
sure